src/main/webapp/javascript/common.js

```diff
@@ -0,0 +1,3 @@
+function goBack() {
+    location.href = "/main.jsp";
```

Comment:
There is a mismatch between the function name here and in other files ("Go back") and actual implementation here, which behaves more like "Go to main page". Please consider using `window.history.back()` instead, or rename the function.

In case you need more information, see:
https://developer.mozilla.org/en-US/docs/Web/API/History_API (overview and examples)
https://developer.mozilla.org/en-US/docs/Web/API/Window/history (API documentation)